### DISCUSSION TITLE
Accounts module display related accounts

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#16779] [How to] [answered]
Hi,

We have added a reseller type section on the accounts. How can we display all customers are related too on its page?

So at the moment we have linked the reseller account to the customers but we can't see the accounts related to the reseller.

We have setup a type field called reseller. Is there a way to show all customers on the reseller account page?

**Comment by apcloic** [comment]
@alextovey 
Hi, 
Your question is not really clear, you should attach screenshot or [screen record](https://www.screentogif.com/).
Regards,

**Comment by MonsterITServices** [comment, marked as answer]
@apcloic 
Hi, Sorry about that.
Customer Account: This shows who the reseller is for this account.
![image](https://user-images.githubusercontent.com/22075161/210323664-d454815a-281e-4dee-9dfb-7f19c0ff07c7.png)

Reseller Account: How can I show all the customer accounts attached?
![image](https://user-images.githubusercontent.com/22075161/210323476-d778491a-3f8e-469f-9566-db94251919ad.png)

Basically I want to show all the accounts linked to the reseller account. Like how the customer accounts show that they have a reseller assigned to them.

**Reply by apcloic** [reply]
Hi,
'Reseller name' field is a new 1:M Related field (account to account) that you have created or is it 'Member of' field that you have renamed ?

**Reply by MonsterITServices** [reply]
HI, Member of field i've renamed.

**Reply by apcloic** [reply]
'Member of' field is a special [UIType](https://doc.yetiforce.com/code/packages/UIType.html) (UIType 51 instead of UIType 10 like other standard 1:M) so I don't think it's a good idea to use it for that purpose.

I think you should better try to create a new 1:M Relation field which will be related to Accounts module.

This way, you should see a new related 'Accounts' tab in Accounts module, you can rename it 'Customers' later if you want.

You may also create a little widget in order to see Customers related to reseller on summary view.

If you need to find customers without resellers, you may use related records filtering mechanism #15201


![account_to_account](https://user-images.githubusercontent.com/2930560/210376323-c90b496a-90cf-4978-923f-f98961bbf52e.gif)

**Reply by MonsterITServices** [reply]
Fantastic just what I wanted. :)
